docs/version0.30.md

```diff
@@ -60,6 +61,32 @@ would yield the following in the <tt>ENVINFO</tt> section after the complete lis
 
 This option specifies/unspecifies classes of objects that will be hashed and extended with a hash slot upon object creation or first move. This option may improve performance for applications that frequently hash objects of a certain type. See [-XX:[+|-]EnsureHashed](xxensurehashed.md) for more details.
 
+### Redesigned heap resizing for balanced GC policy
+
+Heap resizing heuristics have been redesigned for balanced GC. This includes both total heap resizing including Eden and non-Eden components indenpendantly, and also balancing between these two components when heap is fully expanded. The heuristics now combine both CPU overhead (for both Partial GCs and Global Mark Phase) and heap occupancy criteria. The balancing between Eden and non-Eden for fully expanded heaps is far more dynamic (instead of being mostly fixed to ratio 1:4).
+
+As a consequence there should typically be less need for heap sizing tuning options, most notably for Eden sizing options [-Xmn/mns/mnx](xmn.md). 
+
+Also, a new soft limit pause target is added for Partial GCs, that defaults to 200ms. This criteria is combined with PGC CPU overhead critera for a balanced compromise between minimizing footprint, maximizing throughput and meeting the paused time target.
+
+More details about the new heurstics can be found at:
+
+[https://blog.openj9.org/2021/09/24/balanced-gc-performance-improvements-eden-heap-sizing-improvements/](https://blog.openj9.org/2021/09/24/balanced-gc-performance-improvements-eden-heap-sizing-improvements/)
+
+The heuristcs now obey existing options
+
+[-Xmint/-Xmaxt](xmint.md)
```

Comment:
The formatting still isn't quite right as it puts the two options on the same line. Pls try adding `\` to the end of this line.

Reply:
The `\` worked in the markdown, but not on the website. Pls try `<br \>` instead.

Reply:
Sorry, it's a forward slash `<br />`. Maybe we should make them point form by prefixing `-` on the two lines instead. You can pick which you prefer.

Reply:
Connected them with an 'and'. Since there is another list with just one option just below, it would create visaul asymmetry that one list has '-' and another does. 
If this does not work or you don't like it, I can go with '-', but specify if you want the other one-item list to use it too, and if they should be preceded with ':' in the line ahead.

Reply:
It's fine with me as long as it works on the website, and this update does work https://eclipse-openj9.github.io/openj9-docs-staging/891/version0.30/
I only suggested the bullets because we know how to do that, instead of the multiple attempts to get the newline working.

Reply:
With the `and` we probably don't even need the newline any more, but I won't ask to change it again.